src/PluginManager/Microsoft.Performance.Toolkit.PluginManager.Core/Packaging/Metadata/PluginMetadata.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Performance.Toolkit.PluginManager.Core.Packaging.Metadata
+{
+    /// <summary>
+    /// Represents the metadata of a plugin
+    /// </summary>
+    public sealed class PluginMetadata
+    {
+        /// <summary>
+        /// The group this plugin belongs to
+        /// </summary>
+        public string Group { get; set; }
+
+        /// <summary>
+        /// The identifer of this plugin
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// The version of the plugin
+        /// </summary>
+        public Version Version { get; set; }
+
+        /// <summary>
+        /// The human-readable name of this plugin
+        /// </summary>
+        public string DisplayName { get; set; }
+
+        /// <summary>
+        /// A user friendly description of this plugin
+        /// </summary>
+        public string Description { get; set; }
+
+        /// <summary>
+        /// The authors of this plugin
+        /// </summary>
+        public IEnumerable<string> Authors { get; set; }
```

Comment:
I'm wondering if these should be some `PluginAuthor` type that has stuff like contact info on it or something. What do you think?

Reply:
Added